reset-environment/action.yml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+name: 'reset-environment'
+description: 'Power Platform Admin Reset Environment'
```

Comment:
remove "Admin" from the description; it's just an implementation that these verbs are on/in the admin noun/module